shared/wallets/common/buttons/index.js

```diff
@@ -0,0 +1,137 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+
+type SendProps = {|
+  disabled: boolean,
+  disabledDueToMobileOnly: boolean,
+  onSendToKeybaseUser: () => void,
+  onSendToStellarAddress: () => void,
+  onSendToAnotherAccount: () => void,
+  small?: boolean,
+|}
+
+class _SendButton extends React.PureComponent<SendProps & Kb.OverlayParentProps> {
+  _menuItems = [
+    {
+      onClick: () => this.props.onSendToKeybaseUser(),
```

Comment:
don't need new closures here, can just be `this.props.onSendToKeybaseUser` same below